src/turn.ts

```diff
@@ -0,0 +1,47 @@
+import { default as twilio } from "twilio";
+
+/**
+ * Twilio ice server
+ */
+export interface ITwilioIceServer {
+    /**
+     * The STUN/TURN endpoint url
+     */
+    url: string;
+
+    /**
+     * The username required for authentication (TURN endpoint only)
+     */
+    username?: string;
+
+    /**
+     * The credential required for authentication (TURN endpoint only)
+     */
+    credential?: string;
+}
+
+/**
+ * Request TURN servers from Twilio
+ * @param accountSid Twilio AccountSid required for Traversal Service Token retrieval
+ * @param authToken Twilio AuthToken required for Traversal Service Token retrieval
+ * @returns {RTCIceServer[]} the array of TURN servers
+ */
+export const requestTwilioTurnServer = async (accountSid: string, authToken: string) => {
+    const client = twilio(accountSid, authToken);
+    const token = await client.tokens.create();
+    const twilioIceServers = JSON.parse(JSON.stringify(token.iceServers));
+    const iceServers: RTCIceServer[] = [];
+    twilioIceServers.forEach((twilioIceServer: ITwilioIceServer) => {
+        // Filter out turn servers

```

Comment:
any reason we don't choose to simply use all values? In my opinion this is:
1) likely what the user expects anyway
2) cleaner

Reply:
I think that you're right here. It's better to use all.